contrib/mixin/Makefile

```diff
@@ -0,0 +1,16 @@
+.PHONY: all tools manifests test
+
+all: manifests
+
+tools:
+	go install github.com/google/go-jsonnet/cmd/jsonnet@latest
+	go install github.com/brancz/gojsontoyaml@latest
+
+manifests:
+	rm -rf manifests/*.yaml
+	mkdir -p manifests
+	jsonnet -e '(import "mixin.libsonnet").prometheusAlerts' | gojsontoyaml > manifests/etcd-prometheusRules.yaml
+
+test:
```

Comment:
tests should require `manifests/etcd-prometheusRules.yaml`

Reply:
Adjusted it, thanks!